### ISSUE TITLE
Am I overlooking an option?

### ISSUE THREAD

**Issue by GuardianMajor** [uBlockOrigin/uBlock-issues#2174]
### Prerequisites

- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue
- [X] The issue is not present after wholly disabling uBlock Origin ("uBO") in the browser
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] using a new, unmodified browser profile

### Description

Hi Raymond,

I was just wondering if you were aware of this option (uiFlavor) https://github.com/gorhill/uBlock/wiki/Advanced-settings#uiflavor not being in the advanced settings, wasn't sure if it was deprecated and not noted or somehow it was omitted from the release version? (I have 1.43 and this was supposed to be in since 1.25)

Just an FYI. Thanks.

### A specific URL where the issue occurs

https://github.com/gorhill/uBlock/wiki/Advanced-settings#uiflavor

### Steps to Reproduce

1. Open dashboard
2. Select settings tab
3. Click on Advanced settings
4. no `uiFlavor` option

### Expected behavior

Option there to set.

### Actual behavior

No option there, didn't want to manually add if it was intended to not be there.

### Configuration

<details>

```yaml
uBlock Origin: 1.43.0
Chromium: 103
filterset (summary): 
  network: 154875
  cosmetic: 116326
  scriptlet: 34673
  html: 0
listset (total-discarded, last updated): 
  added: 
    adguard-generic: 65726-666, 3d.18h.49m
    adguard-mobile: 7951-42, 3d.18h.48m
    adguard-spyware: 25412-655, 3d.18h.47m
    adguard-spyware-url: 674-13, 3d.18h.46m
    block-lan: 43-0, 8d.1h.16m
    curben-phishing: 29306-0, 4h.6m
    curben-pup: 199-1, 4h.5m
    dpollock-0: 10407-238, 8d.1h.16m
    fanboy-cookiemonster: 35989-94, 8d.1h.16m
    ublock-annoyances: 4488-80, 3d.18h.43m
  default: 
    user-filters: 779-0, never
    easylist: 68680-1233, 3d.18h.45m
    easyprivacy: 27403-10103, 3d.18h.44m
    plowe-0: 3646-1307, 8d.1h.16m
    ublock-abuse: 75-0, 8d.1h.16m
    ublock-badware: 4223-321, 3d.18h.42m
    ublock-filters: 32191-3442, 3d.18h.41m
    ublock-privacy: 225-42, 8d.1h.16m
    ublock-quick-fixes: 227-26, 4h.4m
    ublock-unbreak: 1834-98, 3d.18h.40m
    urlhaus-1: 4970-1, 4h.3m
filterset (user): [array of 779 redacted]
trustedset: 
  added: [array of 12 redacted]
  removed: 
    moz-extension-scheme
    opera-scheme
    vivaldi-scheme
    wyciwyg-scheme
switchRuleset: 
  added: [array of 9 redacted]
  removed: 
    no-large-media: behind-the-scene false
hostRuleset: 
  added: [array of 9 redacted]
  removed: 
modifiedUserSettings: 
  advancedUserEnabled: true
  suspendUntilListsAreLoaded: true
  webrtcIPAddressHidden: true
modifiedHiddenSettings: 
  allowGenericProceduralFilters: true
  autoCommentFilterTemplate: {{date}} | {{origin}}
  autoUpdateDelayAfterLaunch: 60
  cacheStorageAPI: indexedDB
  cnameIgnoreList: false
  cnameIgnore1stParty: false
  cnameIgnoreExceptions: false
  cnameIgnoreRootDocument: false
  cnameReplayFullURL: true
  cnameUncloakProxied: true
  extensionUpdateForceReload: true
  filterAuthorMode: true
  filterOnHeaders: true
  manualUpdateAssetFetchPeriod: 1000
  popupFontSize: 10pt
  selfieAfter: 5
  strictBlockingBypassDuration: 300
  uiStyles: font-family: Arial; font-size: 9pt
  updateAssetBypassBrowserCache: true
supportStats: 
  allReadyAfter: 2115 ms (selfie)
  maxAssetCacheWait: 1473 ms
```
</details>

**Comment by krystian3w** [comment]
Died with remove classic layout.

https://github.com/gorhill/uBlock/releases/tag/1.43.0

> - [Remove support for deprecated "classic" popup panel](https://github.com/gorhill/uBlock/commit/cef36518edd4d6ca4b3278493cb93329b0833408)

**Comment by GuardianMajor** [comment]
Ok, thank you but nothing you have said explains or answers the question posted, so I am going to hope that the dev or someone with more knowledge of what's going on to provide context.

I know that when things are formally "deprecated", they are properly noted in the documentation and nothing in there states your claim that it has "died", so I feel there might be more to it.

**Comment by gwarser** [comment]
https://github.com/uBlockOrigin/uBlock-issues/wiki/Advanced-settings/_compare/44d7b45e87e61067b334380413044161b4065649

**Comment by gorhill** [comment]
Unnecessary comment. If no volunteers cared there wouldn't be a continuously maintained project in the first place. Don't confuse  _"People don't care"_ with _"People don't do what I want them to do"_.

**Comment by GuardianMajor** [comment]
> In Gorhill opinion almost everyone read change log in releases/at reddit.

Yes and I agree wholeheartedly and over 3 decades I have tried to do that. Not an excuse but my schedule has been really hectic lately and haven't had as much time to go through all changes in depth; so I usually skim notes and I must have missed this one. That's why I occasionally go through the settings and compare to the docs to see if anything has changed or new context has been added so I can adjust accordingly. That's the only reason I noticed.
